pkg/yurthub/network/dummyif_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build linux
```

Comment:
why add this comment?

Reply:
After coding, to format my code, so I execute gofmt -w .,then there is an extra line(go:build linux) in some files located under the path pkg/uitl/iptables. But I found that there is no such line in the kubernetes codekubernetes, so i delete this extre line(go:build linux) in those files, forgot to delete this line in pkg/yurthub/network/dummyif_test.go. I think this is caused by gofmt.